src/modules/game_id.js

```diff
@@ -0,0 +1,14 @@
+// async function createApp() {
+//     const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/`;
+//     const response = await fetch(url, {
+//       method: "POST",
+//     });
+//       const data = await response.text();
+//       return data;
+//   }
+//   const appId = await createApp();
+//   console.log(appId)
+
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉